stacks/cli/src/commands/pull.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  CommandLineOptions,
+  CommandMetadata,
+  CommandMetadataInput,
+  CommandMetadataOption,
+  CommandLineInputs,
+  validators
+} from "@ionic/cli-framework";
+import {Command} from "../interface";
+import * as authentication from "../authentication";
+import * as request from "../request";
+import * as utilities from "../utilities";
+import * as formatter from "../formatter";
+import {Function, Asset} from "../interface";
+import * as path from "path";
+import * as yaml from "yaml";
+
+export class PullCommand extends Command {
+  async getMetadata(): Promise<CommandMetadata<CommandMetadataInput, CommandMetadataOption>> {
+    return {
+      name: "pull",
+      summary: "Pull assets from the server.",
+      inputs: [
+        {
+          name: "Output folder",
+          summary: "Name of output folder to store assets.",
```

Comment:
The article is missing here.
This message should be `Name of the output folder to store assets.`